ASHRAE 55 Tables/process_data_ashrae.py

```diff
@@ -0,0 +1,111 @@
+"""
+This code is used to calculate the values for the ASHREA 55 2017 standard.
+This code uses as input the results from the MRT tool. In order to enable this feature, please set the variable output_data_for_ASHRAE_table to true.
+Each time the code will run the MRT will output four files, each containing the following parameters: x, y, z (coordinates) and results of calculations (e.g. PMV)
+The files will generally be saved automatically in the download folder. Copy and paste them in the directory ASHRAE 55 Tables/Output_MRT_tool.
+Then run the code below
+
+@author Federico Tartarini <federico.tartarini@bears-berkeley.sg>
+"""
+
+import pandas as pd
+import glob
+import os
+import matplotlib.pyplot as plt
+from pprint import pprint
+import numpy as np
+
+
+def interpolation(data_frame, value=7.5, axis='x', variable='PMV'):
+    """This functions interpolates between two array, it is used to calculate the value of PMV in the center of the room"""
+    unique_values = data_frame[axis].unique()  # get the coordinates in the axis
+    upper = [x for x in unique_values if x > value][0]  # find the closest values to the one selected
+    lower = [x for x in unique_values if x < value][-1]
+    array_upper = data_frame.loc[data_frame[axis] == upper, variable]  # extract the two arrays that will be used to calculate the interpolated values
+    array_lower = data_frame.loc[data_frame[axis] == lower, variable]
+    result = []
+    for y_low, y_up in zip(array_lower, array_upper):
+        result.append((value - upper) / (lower - upper) * (y_low - y_up) + y_up)
+
+    return result  # return results of interpolation
+
+
+def find_min_distance(data_frame, value=0.5, axis='distance', variable='PMV'):
+    """This functions finds the minimum distance from the window by interpolating between values exported from MRT tool"""
+    unique_values = data_frame[variable].unique()  # get the coordinates in the axis
+    upper = [x for x in unique_values if x > value][0]  # find the closest values to the one selected
+    lower = [x for x in unique_values if x < value][-1]
+    closest_distance = data_frame.loc[data_frame[variable] == upper, axis].values[0]
+    further_distance = data_frame.loc[data_frame[variable] == lower, axis].values[0]
+    array_distances = np.linspace(closest_distance, further_distance, 100)
+    result = []
+    for distance in array_distances:
+        result.append([distance, (distance - closest_distance) / (further_distance - closest_distance) * (lower - upper) + upper])
+
+    _ = pd.DataFrame(result, columns=[axis, variable])
+    return _.loc[_[variable] <= 0.5, axis].min()  # return results of interpolation
+
+
+plt.close('all')
+
+# directory containing input files
+dir_files = os.path.join(os.getcwd(), 'ASHRAE 55 Tables', 'Output MRT tool')
```

Comment:
It looks like we're assuming this script is run from the root directory of the repo?